substrate/utils/wasm-builder/src/wasm_project.rs

```diff
@@ -910,7 +919,10 @@ fn build_bloaty_blob(
 	if crate::build_std_required() {
```

Comment:
We always need to build core,allocation for RISCV since it isn't shipped with rustup. But we guard it behind this conditional.

Reply:
So, what I need to do here?

Reply:
The way it currently is is fine. :P The `build-std` will be always enabled by default for RISC-V anyway.

Reply:
OK resolving this one then!